governance/remote_executor/Cargo.toml

```diff
@@ -6,3 +6,6 @@ members = [
 
 [profile.release]
 overflow-checks = true
+
+[patch.crates-io]
+serde_wormhole = { git = "https://github.com/wormhole-foundation/wormhole", tag = "v2.17.1" }
```

Comment:
This patch.crate-io wormhole thing is getting out of control, wormhole needs to fix

Reply:
Agreed, reasonable fix for now though. Let's ask for publishing.